### ISSUE TITLE
BlazorBoilerplate.Client Fails to load

### ISSUE THREAD

**Issue by EmergentCybernetics** [enkodellc/blazorboilerplate#555]
I'm getting the following error in my browser when I try to run `BlazorBoilerplate.Client` in debug or release mode. I'm using the `master` branch from today. Any idea what's causing this? I have .Net 5.0 and ASP.Net Core 5.0 SDKs and runtimes installed. Running `BlazorBoilerplate.Server` in debug or release mode works fine.

### Common solutions to this issue:
**The specified version of Microsoft.NetCore.App or Microsoft.AspNetCore.App was not found.**

### Specific error detected by ANCM:
Cannot use file stream for [C:\Users\MyUser\Documents\Development\blazorboilerplate\src\Client\BlazorBoilerplate.Client\bin\Debug\net5.0\BlazorBoilerplate.Client.deps.json]: No such file or directory A fatal error was encountered. The library 'hostpolicy.dll' required to execute the application was not found in 'C:\Users\MyUser\Documents\Development\blazorboilerplate\src\Client\BlazorBoilerplate.Client\bin\Debug\net5.0'. Failed to run as a self-contained app. - The application was run as a self-contained app because 'C:\Users\MyUser\Documents\Development\blazorboilerplate\src\Client\BlazorBoilerplate.Client\bin\Debug\net5.0\BlazorBoilerplate.Client.runtimeconfig.json' was not found. - If this should be a framework-dependent app, add the 'C:\Users\MyUser\Documents\Development\blazorboilerplate\src\Client\BlazorBoilerplate.Client\bin\Debug\net5.0\BlazorBoilerplate.Client.runtimeconfig.json' file and specify the appropriate framework.

### Troubleshooting steps:

- Check the system event log for error messages
- Enable logging the application process' stdout messages
- Attach a debugger to the application process and inspect

### For more information visit: https://go.microsoft.com/fwlink/?LinkID=2028526

**Comment by enkodellc** [comment]
@EmergentCybernetics - sorry you are having issues. Please review the documentation:  https://blazor-boilerplate.readthedocs.io/en/latest/features/dual_mode_blazor.html

You will still run server but you will change the the settings once you log in as admin to switch to WebAssembly.  

You could hard code it to start with WebAssembly from the TenantSettingValues.cs  
`     { SettingKey.MainConfiguration_Runtime,         new Tuple<string, SettingType>(BlazorRuntime.Server.ToString(), SettingType.String) },`

**Comment by EmergentCybernetics** [comment]
@enkodellc 

Thanks for the quick response!

I saw this in the README, `Just set the desidered (spelling??) runtime in admin section.`, but wasn't sure where that setting was. I was looking in the `appsettings.json`, not the UI.

The `Documentation` link is kind of buried in the README. I didn't even realize you had docs. I'm reviewing those now.

I switched the setting in the UI and it is loading the WebAssembly build now. Thanks again!
